### ISSUE TITLE
"Awesome-azd", "Get started", "Contribute", "Docs", "Resources" controls are links but not associated with blue color and underline/chevron to differentiate with normal text: A11y_Awesome AZD_Landing page_Top navigation_Usability.

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#337]
### Tags
#A11yTCS; #A11ySev4; #A11yUsable; #WCP; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #Feature_AwesomeAZD_Web_Mar2024; #DesktopWeb;

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20 
Windows 11 Enterprise.
Version 23H2 OS build 22631.2050.
### Repro Steps:
1. Hit this URL: https://azure.github.io/awesome-azd/.
2. Tab till “Top navigation” controls.
3. Now verify the issue that "Awesome-azd", "Get started", "Contribute", "Docs", "Resources" controls are links but not associated with blue color and underline/chevron to differentiate with normal text or not.
### Actual Result:
"Awesome-azd", "Get started", "Contribute", "Docs", "Resources" controls are links but not associated with blue color and underline/chevron to differentiate with normal text.
**Note: Facing similar issue in the following scenarios.**
1. Filter Results--Viewing templates.
2. Landing page-- Footer
### Expected Result:
"Awesome-azd", "Get started", "Contribute", "Docs", "Resources" controls should be associated with blue color and underline/chevron to differentiate with normal text.
### User Impact:
Low vision users will face difficulty to identify the links with normal text if links are not associated with underline/Chevron.

### Attachment:

<img width="960" alt="337_A11y_Awesome AZD_Landing page_Top navigation_Usability" src="https://github.com/Azure/awesome-azd/assets/100270074/f7ac0eba-85b6-4f89-8dd8-7fe41e5e2ebf">
<img width="960" alt="337_A11y_Awesome AZD_Filter Results_Viewing templates_Similar issue" src="https://github.com/Azure/awesome-azd/assets/100270074/33249f5d-bdfb-4c3a-a7b0-2d803cb7cd6a">
<img width="960" alt="337_A11y_Similar issue" src="https://github.com/Azure/awesome-azd/assets/100270074/1429bc61-b2c4-4a31-9d12-16e2134592ac">

**Comment by hemarina** [comment]
Closing this issue as we made a whole update on the site, and this issue is not update to date with existing site. @Harika1702 Feel free to reopen this issue if needed.
